src/modules/git/index.ts

```diff
@@ -37,6 +37,7 @@ export class GitModule {
    * 'LF' = '\n',
    * 'CRLF' = '\r\n'
    *
+   * @param options.refDate The date to use as reference point for the commit. Defaults to now.
```

Comment:
I'm not sure why it doesn't show a hint here, but IMO the empty line should be between the param and example block:

```suggestion
   * @param options.refDate The date to use as reference point for the commit. Defaults to now.
   *
```